examples/platforms/jn5189/Makefile.platform.am

```diff
@@ -0,0 +1,44 @@
+#
+#  Copyright (c) 2019, The OpenThread Authors.
+#  Copyright (c) 2019, NXP.
```

Comment:
The only copyright holder should be "The OpenThread Authors"

Reply:
Fixed.